docs/contributors/code/coding-guidelines.md

```diff
@@ -116,7 +116,10 @@ import VisualEditor from '../visual-editor';
 
 ### Experimental and Private APIs
 
-Experimental APIs are temporary values exported from a module whose existence is either pending future revision or provides an immediate means to an end.
+There are two important API types in the Gutenberg codebase:
+
+* **Experimental APIs** – they are public APIs that are only shipped in Gutenberg and not merged into WordPress core. Their existence is either pending future revision or provides an immediate means to an end.
```

Comment:
Personally, I think that experimental APIs that are only meant for the Gutenberg plugin are useless. I think we can omit them.

Reply:
Oh! I thought they’re a good way of getting extenders feedback for APIs that are not mature yet?

Reply:
The thing I'm not sure that we need anything for that since the Gutenberg plugins is an alpha version. In other words, if we want to ship something into core, there's not specific to be done and if we want to limit something for the Gutenberg plugin for some time, we use IS_GUTENBERG_PLUGIN

In other words, I'm not sure we really need to formalize "experimental APIs" unless there's a way to ship these into core.

Reply:
Oh I see! Yeah `IS_GUTENBERG_PLUGIN` is key. I guess the only value in having an `__experimental` prefix is that you can find out which APIs may not be in core just by looking at the code. Personally I like that, but I'm also happy to remove it from the docs.

Reply:
@youknowriad I updated the document – what do you think now?